project/ROADMAP.md

```diff
@@ -0,0 +1,5 @@
+Galaxy Project Roadmap
```

Comment:
The contents of this file I would rather see as a combination of GH Issues and Milestones than an extra file we need to keep updating.